From d488de23b75b2f3e235f1c184b2253d1402b4c0f Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 10 May 2018 23:47:56 -0700
Subject: [PATCH] glib/CMakeLists.txt: Add libpoppler to link along with
 poppler-glib

This is required since poppler-glib uses symbols from poppler and
we use --as-needed

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 glib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glib/CMakeLists.txt b/glib/CMakeLists.txt
index 33c6608..87e1b44 100644
--- a/glib/CMakeLists.txt
+++ b/glib/CMakeLists.txt
@@ -127,7 +127,7 @@ if (HAVE_INTROSPECTION AND BUILD_SHARED_LIBS)
   get_directory_property(_tmp_includes INCLUDE_DIRECTORIES)
   _list_prefix(_includes _tmp_includes "-I")
   set(Poppler_0_18_gir_CFLAGS ${_includes} -L${CMAKE_BINARY_DIR})
-  set(Poppler_0_18_gir_LIBS poppler-glib)
+  set(Poppler_0_18_gir_LIBS poppler-glib poppler)
   _list_prefix(_abs_introspection_files introspection_files "${CMAKE_CURRENT_SOURCE_DIR}/")
   list(APPEND _abs_introspection_files
     ${CMAKE_CURRENT_BINARY_DIR}/poppler-enums.c
